greg/aux_functions.py

```diff
@@ -196,8 +210,12 @@ def tag(placeholders):
     file_to_tag = eyed3.load(podpath)
     for mytag in tagdict:
         setattr(file_to_tag.tag, mytag, tagdict[mytag])
+    if coverart:
+        with open(coverart_filename, 'rb') as imagefile:
```

Comment:
It would be cool if the `coverart_filename` could be a http(s) URL. And if the images linked from the feed (e.g. `<image><url>` in the feed and `<itunes:image>` in the article) would be made available as placeholders. This way we could do:

```
coverart = {articleimage}
```

or

```
coverart = {feedimage}
```

Reply:
Right, let me look into that.